daprdocs/content/en/reference/components-reference/supported-bindings/gcpbucket.md

```diff
@@ -316,6 +387,58 @@ The list of objects will be returned as JSON array in the following form:
 	}
 ]
 ```
+
+### Copy objects
+
+To perform a copy object operation, invoke the GCP bucket binding with a `POST` method and the following JSON body:
+
+```json
+{
+  "operation": "copy",
+  "metadata": {
+    "destinationBucket": "destination-bucket-name",
+  }
+}
+```
```

Comment:
Do we need specific responses on any of these? Assuming 204 no content for success or?

Reply:
maybe just link to the output binding ref: https://docs.dapr.io/reference/api/bindings_api/#http-response-codes-2

Reply:
added response sections!